client/src/services/icecreams.ts

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+const iceCreamUrl = "http://localhost:3003/api/icecreams/suggestions/5";
+const phraseUrl = "http://localhost:3003/api/words/";
+
+const getFive = () => {
+  const request = axios.get(iceCreamUrl);
```

Comment:
We should probably do a little error handling here, just to be safe
https://javascript.info/try-catch

Reply:
Added axios's own catches for this!